actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,33 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
```

Comment:
What about to introduce `ArrayInquirer` to `ActiveSupport`? There is already [similar code](http://guides.rubyonrails.org/active_support_core_extensions.html#inquiry) for `String`.